fish_prompt.fish

```diff
@@ -407,6 +406,17 @@ function __bobthefish_prompt_rubies -d 'Display current Ruby (rvm/rbenv)'
   set_color normal
 end
 
+function __bobthefish_prompt_rubies -d 'Display current Ruby information'
+  switch "$theme_display_ruby"
+    case no; return
+    case auto
```

Comment:
I still think this is wrong. If you have this:

```
foo/
foo/.ruby-version
foo/bar/
```

… and you're currently inside `foo/bar/`, your ruby version is affected by the `.ruby-version` in parent. So not showing the custom ruby version when you're inside a subfolder is wrong.